docker/Dockerfile

```diff
@@ -44,10 +44,12 @@ WORKDIR /var/www
 
 RUN curl -o wait-for.sh 'https://raw.githubusercontent.com/eficode/wait-for/v2.2.3/wait-for'
 RUN composer install --no-interaction --prefer-dist --optimize-autoloader
+RUN composer require guzzlehttp/psr7:2.4.5
```

Comment:
Why has this been added? What does it do?

Reply:
`guzzlehttp/psr7` is a package that serves as a dependency for `guzzlehttp/guzzle`, as defined in our composer.json file. Unfortunately, `guzzlehttp/psr7` contains several high-level vulnerabilities. We're unable to specify a version for it in composer.json because it conflicts with the requirements of `guzzlehttp/guzzle`. Running the provided command is the most straightforward solution to address this issue.